package/Dockerfile.agent

```diff
@@ -1,18 +1,25 @@
 ARG RANCHER_TAG=dev
 ARG RANCHER_REPO=rancher
+ARG BASE_DISTRIBUTION=registry.suse.com/suse/sle15:15.3
 FROM ${RANCHER_REPO}/rancher:${RANCHER_TAG} as rancher
 
-FROM ubuntu:18.04
```

Comment:
note to self: we need to understand why this remained on 18.04 while others were 20.04